nixos/modules/services/networking/xrdp.nix

```diff
@@ -103,9 +105,21 @@ in
         defaultText = literalMD "generated from configuration";
         description = lib.mdDoc "The location of the config files for xrdp.";
       };
+
+      extraConfDirCommands = mkOption {
+        type = types.str;
+        default = "";
+        description = lib.mdDoc "Extra commands to run on the default confDir constructor";
+      };
```

Comment:
Can you add an example when people would want to use this?

Reply:
Well, the default options often works but what if you want to change something? The module is not RFC42 compliant so it's the least inconvenient way I found.

Reply:
Would be nice to have an example though, so people can know what type of settings can be set here.

Reply:
I added an example that I was using to test some stuff, and an extra replace to show how another setting can be changed